src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int NUMBER_OF_FIGURES = 5;
+    private final String DEFAULT_COLOR = "WHITE";
+    private final int DEFAULT_RADIUS = 10;
+    private final int MAX_BOUND = 50;
+    private final Random random = new Random();
+    private final ColorSupplier supplier = new ColorSupplier();
+    String randomColor = supplier.getRandomColor();
```

Comment:
with such approach you "random" color will be generated only once when instance of FigureSupplier is created and is not going to change after it. So it's not really random. Please move this to method so value would be regenerated for each call for new figure